biz.aQute.repository/src/aQute/bnd/repository/maven/provider/MavenBndRepository.java

```diff
@@ -249,6 +249,12 @@ private Resource getPom(PutOptions options, ReleaseDTO instructions, Jar binary)
 					.orElse(null);
 			} else {
 				pom = createPomFromFile(options.context.getFile(instructions.pom.path));
+				if (pom == null) {
+					logger.warn(
+						"A pom path was set in the -maven-release instuction, but no file could be found in {} (base path: {}) ",
+						instructions.pom.path, options.context.getBaseURI()
```

Comment:
I think the 2nd arg should just be `options.context.getBase()`. I don't see why we would want to use a URI here.